### PR TITLE
Certora GSM Review

### PR DESCRIPTION
**Pull request by MichaelMorami**

### CODE REVIEW

File: .github/workflows/certora-gsm.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+name: certora-GSM
```
Comment (miguelmtzinf) [inline .github/workflows/certora-gsm.yml:1]:
```suggestion
name: certora-gsm
```

File: certora/GSM/conf/non-4626/Dominik-gho-assetToGhoInvertibility.conf [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        "node_modules/@openzeppelin/contracts/utils/math/Math.sol"
+    ],
+    "link": [
+    "GsmHarness:GHO_TOKEN=GhoToken",
+    "GsmHarness:PRICE_STRATEGY=FixedPriceStrategyHarness",
+    "GsmHarness:_feeStrategy=FixedFeeStrategyHarness"
+    ],
+//    "rule" : [
+//       "buyAssetInverse_all","sellAssetInverse_all"
+//    ],
```
Comment (miguelmtzinf) [inline certora/GSM/conf/non-4626/Dominik-gho-assetToGhoInvertibility.conf:19]: is this intended?

File: certora/GSM/conf/non-4626/fees-sell.conf [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        "-adaptiveSolverConfig false",
+        "-smt_nonLinearArithmetic true",
+    ],
+    "verify":
+        "GsmHarness:certora/GSM/specs/gsm/fees-sell.spec",
+//        "GsmHarness:certora/GSM/specs/gsm/fees-sell-R3.spec",
+//    "rule": [
+//        "R3_10",
+//        "R3_11",
+//    ],
```
Comment (miguelmtzinf) [inline certora/GSM/conf/non-4626/fees-sell.conf:38]: same question here

File: certora/GSM/conf/non-4626/otakar-FixedFeeStrategy.conf [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    "solc": "solc8.10",
+    "rule_sanity" : "basic",
+    "msg": "Otakar: FixedFeeStrategy",
+    "multi_assert_check": true,
+    "smt_timeout": "4000",
+    "prover_args": [
+//        "-copyLoopUnroll 6",
+        "-depth 20",
+//        "-newSplitParallel true",
+//        "-smt_hashingScheme plainInjectivity",
```
Comment (miguelmtzinf) [inline certora/GSM/conf/non-4626/otakar-FixedFeeStrategy.conf:22]: here too

File: certora/GSM/mutations/mutants/Gsm/Gsm.sol0 copy.sol
Comment (miguelmtzinf) [inline certora/GSM/mutations/mutants/Gsm/Gsm.sol0 copy.sol:1]: is this a back up?

File: certora/GSM/mutations/mutants/Gsm/Gsm.sol1 copy.sol
Comment (miguelmtzinf) [inline certora/GSM/mutations/mutants/Gsm/Gsm.sol1 copy.sol:1]: same q here

File: certora/GSM/conf/non-4626/Dominik-gho-assetToGhoInvertibility.conf (hunk 2 above)
Comment (MichaelMorami) [reply certora/GSM/conf/non-4626/Dominik-gho-assetToGhoInvertibility.conf:19]: good catch! absolutely not.

File: certora/GSM/conf/non-4626/fees-sell.conf (hunk 3 above)
Comment (MichaelMorami) [reply certora/GSM/conf/non-4626/fees-sell.conf:38]: done

File: certora/GSM/conf/non-4626/otakar-FixedFeeStrategy.conf (hunk 4 above)
Comment (MichaelMorami) [reply certora/GSM/conf/non-4626/otakar-FixedFeeStrategy.conf:22]: done

File: certora/GSM/mutations/mutants/Gsm/Gsm.sol0 copy.sol
Comment (MichaelMorami) [reply certora/GSM/mutations/mutants/Gsm/Gsm.sol0 copy.sol:1]: this is not a backup, just a very bad naming system which originates from gambit.
I'll make them sequential

**Pull request merged**